tests/testthat/test-pivot-wide.R

```diff
@@ -1,3 +1,29 @@
+test_that("error if input is not a data.frame", {
+  spec <- tibble::tibble(
+    .name = c("x", "y", "z"),
+    .value = "val",
+    key = c("x", "y", "z")
+  )
+
+  expect_no_error(
```

Comment:
Since this test is about erroring when it's not a data frame, I'd suggest dropping this expectation. (And I wouldn't test it elsewhere just because I know that if `pivot_wider_spec` stopped working with data frames so much of tidyr would break)